discord_bot_main.py

```diff
@@ -17,11 +17,16 @@ def __init__(self,*args,**kwargs):
         super().__init__(*args, **kwargs)
 
         self.words_file_path = str()
+        self.wordsmith_words_file_path = str()
         self.words = set()
+        self.wordsmith_words = set()
+        
 
     def loadWords(self):
         with open(self.words_file_path) as f:
             self.words = set(f.read().splitlines())
+        with open(self.words_file_path) as f:
+            self.wordsmith_words = set(f.read())
```

Comment:
If I understand correctly, this will make the set only have one element (The file content str).  I suggest we follow the same pattern as above so the set has one entry per line. That way you can do a quick-lookup to find the word, instead of having to use str.find, which will inspect the entire file string.